democontrollertabs/democontrollertabs.php

```diff
@@ -36,14 +36,14 @@ public function __construct()
         $this->name = 'democontrollertabs';
         $this->author = 'PrestaShop';
         $this->version = '1.0.0';
-        $this->ps_versions_compliancy = ['min' => '1.7.7', 'max' => '8.99.99'];
+        $this->ps_versions_compliancy = ['min' => '9.0.0', 'max' => _PS_VERSION_];
```

Comment:
It's a tiny detail and I didn't notice it in other examples, but having `max` to `9.99.99` is recommended and that's what developers should do

Reply:
You're right, fixed! I'll look into the other modules we might have once we handle them all so we have something homogenous everywhere 😉

Reply:
@kpodemski Why would you do this? You then have to update ALL modules when releasing a new major version. We don't do this on native modules, no 3rd party developer I know does this. Why here?

Reply:
@Hlavtox 

>  You then have to update ALL modules when releasing a new major version.

and that's ok, in the future this should help Update Assistant or other tools to do the list of compatibility issues before upgrading from one major to another

Reply:
@kpodemski I forgot we all have too much free time and money on our hands to add extra work. 😜

I get there is a value, but, is it worth the hassle? :-) Sometimes there is an issue on module branch because of which you can't release a new version and nobody fixes it. Look at GDPR module which is basically dead now.